iset.mm

```diff
@@ -63863,6 +63893,54 @@ all reals (greatest real number) for which all positive reals are greater.
       KVBFVKVDTVEWHXFAWKCXDWGXEDWCWKPVFRVHTVLVM $.
   $}
 
+  ${
+    $d A x y $.  $d B x y $.
+    $( A disjoint union is a subset of a Cartesian product.  (Contributed by
+       AV, 25-Jun-2022.) $)
+    djuss $p |- ( A |_| B ) C_ ( { (/) , 1o } X. ( A u. B ) ) $=
+      ( vx vy c0 c1o cv wcel cinl cfv wceq wrex cinr wa cop cvv cmpt a1i adantr
```

Comment:
Remark: djuex is also a consequence of this, since the RHS is a set. I don't know if this provides a shorter proof.

Reply:
Given that it's sizably shorter, I would go for this second proof. Or if you prefer, keep the first proof and simply mention in its comment "Note that a shorter proof is possible from ~ djuss and ~  ssexd ."

Reply:
I would like to keep the first proof and to provide the second proof as ALT version. Iwould propose, however, to provide this ALT version in set.mm only, not in iset.mm (I do not know exactly how to handle discouraged theorems in iset.mm). @jkingdon what do you think?

Reply:
> Iwould propose, however, to provide this ALT version in set.mm only, not in iset.mm (I do not know exactly how to handle discouraged theorems in iset.mm

The answer to the question about `discouraged` is that there is an `iset-discouraged` which is managed just like its set.mm counterpart.

I guess I'd vote for just having the ALT version in set.mm. In most cases there isn't any reason to have ALT versions in both set.mm and iset.mm.